### PR TITLE
Adjust Play page for keywords

### PR DESCRIPTION
**Pull request by akeller**
## Description

Closes https://github.com/camunda/developer-experience/issues/315.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [ ] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

**Comment by akeller** [comment]
I didn't backport these changes, but I'm also looking for a little help.

I want a crisper comparison sentence between Play and Operate, or put another way, what you can do in Play that you may otherwise do in Operate if you weren't already Play-ing.

**Comment by HanselIdes** [comment]
> I want a crisper comparison sentence between Play and Operate, or put another way, what you can do in Play that you may otherwise do in Operate if you weren't already Play-ing.

Operate is designed to monitor many production process instances and intervene only as necessary, while Play is designed to drive a single process instance through the process and mock external systems.  Both offer monitoring of a single process instance, its variables and path, incidents, as well as actions to modify or repair a process instance. Operate offers bulk actions and guardrails against breaking production processes while Play offers a streamlined UX to quickly run through scenarios.

@akeller does this help?

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 1]
```diff
@@ -61,7 +63,7 @@ After completing part of your process, you can **rewind** to a previous element
 
 ![rewind process](img/play-rewind.png)
 
-The rewind operation currently does not support the following elements:
+The rewind operation in Play currently does not support the following elements:
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:66]:
```suggestion
Play's rewind operation currently does not support the following elements:
```

File: docs/components/modeler/web-modeler/play-your-process.md [hunk 2]
```diff
@@ -45,7 +47,7 @@ You have a few options to mock an external system:
 - In **Implement** mode, hard-code an example payload in the task or event **Output** section.
 - When completing a task or event, use the secondary action to complete it with variables.
 
-Incidents are raised as they would in any Camunda cluster. Use the variables and incident message to debug the issue.
+Incidents are raised as they would in any Camunda cluster, just within the context of Play, instead of Operate. Use the variables and incident message to debug the issue, all while staying in Play.
```
Comment (HanselIdes) [inline docs/components/modeler/web-modeler/play-your-process.md:50]:
```suggestion
Incidents are raised in Play just like in Operate. Use the variables and incident message to debug the process instance.
```

**Review by HanselIdes — COMMENTED** [review]
The blatant keyword-stuffing felt a little disruptive to the documentation's flow, so I made a few recommendations to tone it down

**Comment by akeller** [comment]
@HanselIdes thanks for the collab here! I'll clean this up later today and include the Operate comparison info. I think this will help humans and AI position Play and Operate, but I'll let you know when it's live and ingested so we can test.

**Review by conceptualshark — APPROVED** [review]

**Review by christinaausley — APPROVED** [review]

**Pull request merged**